cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -97,7 +98,9 @@ def test_memory_checkbox(small_test_vm, provider, soft_assert):
 
 
 @pytest.mark.rhv1
-@pytest.mark.meta(automates=[1608475])
+@pytest.mark.rhv3
```

Comment:
These are supposed to be RHV levels for testing (equivalent to tiers), and I don't think it makes sense to have more than one level (tier) marked on a case?